web/app/js/components/Top.jsx

```diff
@@ -144,13 +144,15 @@ class Top extends React.Component {
     });
   }
 
-  updateTapClosingState(isTapClosing) {
+  updateTapClosingState(isTapClosing, isTapRequestInProgress) {
```

Comment:
We only call this from one place, and it's to set both of these to false. We should consider removing the args in this case and just having
```
  updateTapClosingState() {
    this.setState({
      tapIsClosing: false,
      tapRequestInProgress: false
    });
  }
```